e2e/template.go

```diff
@@ -10,15 +10,15 @@ import (
 // TODO(ace): minimize the actual required defaults.
 // this is what we previously used for bash e2e from e2e/nodebootstrapping_template.json.
 // which itself was extracted from baker_test.go logic, which was inherited from aks-engine.
-func baseTemplate() *datamodel.NodeBootstrappingConfiguration {
+func baseTemplate(location string) *datamodel.NodeBootstrappingConfiguration {
 	var (
 		trueConst  = true
 		falseConst = false
 	)
 	return &datamodel.NodeBootstrappingConfiguration{
```

Comment:
we need to do some work on getting this actually completely correct depending on k8s version, cluster fqdn, etc.